src/main/java/duke/exceptions/IncompleteCommandException.java

```diff
@@ -0,0 +1,5 @@
+package duke.exceptions;
+
+public class IncompleteCommandException extends Exception {
+    //no other code needed
+}
```

Comment:
Here (and below), use the opportunity provided by inheritance to include a more expressive error message rather than empty classes